<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <column>
     <property name="text">
      <string>Nazwisko</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wiek</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pkt.</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="dodaj">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>dodaj zawodnika</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wyniki">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sprawd≈∫ wyniki</string>
    </property>
   </widget>
   <widget class="QLabel" name="zawodnik2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="zawodnik1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="zawodnik3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
